nesta/core/luigihacks/misctools.py

```diff
@@ -61,3 +64,64 @@ def find_filepath_from_pathstub(path_stub):
             if path.endswith(path_stub.rstrip("/")):
                 return path
         relative += 1
+
+
+def f3p(path_stub):
+    """Shortened name for coding convenience"""
+    return find_filepath_from_pathstub(path_stub)
+
+
+def load_yaml_from_pathstub(pathstub, filename):
```

Comment:
note: migrated here from `orm_utils` since it's not an ORM util and it's only used in `orm_utils` and `misc_utils`, so migration was straightforward